ui/desktop/electron-app/config/forge.config.js

```diff
@@ -20,14 +20,22 @@ module.exports = {
       'entitlements-inherit': './assets/macos/entitlements.plist',
       'signature-flags': 'library',
     },
-    asar: true,
-  },
-  plugins: [
```

Comment:
It turns out through debugging that this plugin only unpacks `*.node` files but we also need a `spawn-helper` provided by `node-pty`